py/plugins/ollama/src/genkit/plugins/ollama/models.py

```diff
@@ -1,16 +1,22 @@
 # Copyright 2025 Google LLC
 # SPDX-License-Identifier: Apache-2.0
 import logging
+from typing import Literal
 
 from genkit.core.action import ActionRunContext
 from genkit.core.typing import (
     GenerateRequest,
     GenerateResponse,
     GenerateResponseChunk,
     GenerationCommonConfig,
+    Media1,
```

Comment:
please sync to head, these have been fixed, can use `Media` (instead of `Media1`). Same for `ToolRequest1` below.